capabilitytypes/radon.capabilities.datapipeline/ConnectToPipeline/ConnectToPipeline.tosca

```diff
@@ -0,0 +1,9 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
```

Comment:
It seems you are using a wrong filename. It should be `<EntityType>.tosca`. Please see the other types of the particles.

Reply:
Some of the manual fixes to errors introduced in removal of versions caused this. Should be solved now.